docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3"
+
+services:
+  dev-build:
+    image: todo-app:dev
+    build:
+      context: .
+      dockerfile: Dockerfile
+      target: development
+
+  dev-run:
+    image: todo-app:dev
+    env_file:
+      - ./.env
+    ports:
+      - "5000:5000"
+    volumes:
+      - .:/app
+      - todo_app:/app/todo_app
```

Comment:
What's the motivation for having a named `todo_app` volume here as well as the bind mount?